src/editors/containers/EditorContainer/components/TitleHeader/EditableHeader.jsx

```diff
@@ -1,25 +1,26 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-import { Icon, Form } from '@edx/paragon';
-import { Edit } from '@edx/paragon/icons';
+import { Form } from '@edx/paragon';
+import EditConfirmationButtons from './EditConfirmationButtons';
 
 export const EditableHeader = ({
   handleChange,
   updateTitle,
   handleKeyDown,
   inputRef,
   localTitle,
+  cancelEdit,
 }) => (
   <Form.Group>
     <Form.Control
+      style={{ 'padding-inline-end': 'calc(1rem + 84px)' }}
```

Comment:
nit: for next time: https://mdbootstrap.com/docs/react/utilities/spacing/ we use classNames to reference bootstrap styles, in this case. I'm going to say this is ok for now, because this is something which improves performance so minisculely.

Reply:
The padding only allows for up to the value 5 which was not enough for two buttons inside the input. So I had to include this work around.